### PR TITLE
Nova Scotia The Poverty Reduction Credit (PRC)

### LINKED ISSUE #250: Nova Scotia The Poverty Reduction Credit (PRC)
**Reference**: https://novascotia.ca/coms/noteworthy/PovertyReductionCredit.html

### PR DESCRIPTION
**Pull request by ziyuliuzilla**
Fixes #250

### CODE REVIEW

File: policyengine_canada/variables/gov/provinces/ns/tax/income/credits/nsprc/ns_poverty_reduction_credit.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class ns_poverty_reduction_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Nova Scotia Poverty Reduction Credit"
+    unit = CAD
+    definition_period = YEAR
+    reference = (
+        "https://novascotia.ca/coms/noteworthy/PovertyReductionCredit.html"
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/ns/tax/income/credits/nsprc/ns_poverty_reduction_credit.py:11]:
```suggestion
        "https://novascotia.ca/coms/noteworthy/PovertyReductionCredit.html",
        "https://nslegislature.ca/sites/default/files/legc/statutes/income%20tax.pdf#page=121"
```

This legislative reference doesn't define the program but does assign responsibility for defining it to the governor.

File: policyengine_canada/variables/gov/provinces/ns/tax/income/credits/nsprc/ns_poverty_reduction_credit.py
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/ns/tax/income/credits/nsprc/ns_poverty_reduction_credit.py:1]: Please rename the folder to `prc`, also for tests and parameters

In general let's only include the province code in abbreviations if the program is referred to with the province in the title within the province, such as Ontario Child Benefit.

File: policyengine_canada/variables/gov/provinces/ns/tax/income/credits/nsprc/ns_poverty_reduction_credit.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    )
+    defined_for = ProvinceCode.NS
+
+    def formula(household, period, parameters):
+        income = household("adjusted_family_net_income", period)
+        children = household("count_children", period)
+        p = parameters(period).gov.provinces.ns.tax.income.credits.nsprc
+        eligible = (children == 0) & (income < p.income_threshold)
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/ns/tax/income/credits/nsprc/ns_poverty_reduction_credit.py:19]: What about the Income Assistance requirement?

Comment (ziyuliuzilla) [reply policyengine_canada/variables/gov/provinces/ns/tax/income/credits/nsprc/ns_poverty_reduction_credit.py]: Pavel and I have created an empty ns_assistance_requirement variable, and we will create this program in the next pull request

**Pull request merged**